pkg/printers/internalversion/describe_test.go

```diff
@@ -1664,9 +1664,25 @@ Spec:
     From Pod Selector: id=app2,id2=app3
     From Namespace Selector: id=app2,id2=app3
     From Namespace Selector: foo in (bar1,bar2),id=app2,id2=app3
+    From IPBlock:
+        CIDR: 192.168.0.0/16
+        Except: 192.168.3.0/24, 192.168.4.0/24
     ----------
     To Port: <any> (traffic allowed to all ports)
     From: <any> (traffic not restricted by source)
+  Allowing egress traffic:
+    To Port: 80/TCP
+    To Port: 82/TCP
+    To Pod Selector: id=app2,id2=app3
```

Comment:
I know that the previous PR has this same format, but I think we should not split this into two words. Maybe have a follow-up commit to switch field types to PodSelector and NamespaceSelector.